api/resources_portal/models/material_request.py

```diff
@@ -62,3 +59,11 @@ class Meta:
     is_active = models.BooleanField(default=True)
 
     status = models.CharField(max_length=32, choices=STATUS_CHOICES, default="PENDING")
+
+    @computed(
+        models.ForeignKey(
+            User, blank=False, null=False, on_delete=models.CASCADE, related_name="assignments"
+        )
+    )
+    def assigned_to(self):
+        return self.material.contact_user
```

Comment:
I think this should be a field of its own that is set to this by default, but at some point we'll want to let people change it. (For example, someone could be the contact_user for a material, but then hand out fulfillments on a rotating basis or something to share the workload.)